kubetest/BUILD

```diff
@@ -11,6 +11,9 @@ go_binary(
     name = "kubetest",
     importpath = "k8s.io/test-infra/kubetest",
     library = ":go_default_library",
+    x_defs = {
+        "main.BuildTag": "{DOCKER_TAG}",  # keep
```

Comment:
hummm, so where's `DOCKER_TAG` come from? Probably update the comment to be more descriptive

Reply:
this is set by `hack/print-workspace-status.sh`, but it should be `{STABLE_DOCKER_TAG}`, not `{DOCKER_TAG}`.

Reply:
wait, this is test-infra, not k/k. in this case it's just `print-workspace-status.sh`, and this looks fine.